src/containers/Accounts/AccountTransactionsTable/styles.scss

```diff
@@ -164,7 +164,7 @@
     .partial-payment,
     .closed,
     .flag,
-    .unsetregularkey,
+    .unset,
```

Comment:
Why not just have two different `unset`, one for SignerListSet and one for SetRegularKey?

Reply:
There will be more unsets in the future.  For example domains and email hashes on accountset can be cleared out.  We dont current do anything with those.